src/data/set/n_ary.lean

```diff
@@ -67,6 +67,12 @@ lemma forall_image2_iff {p : γ → Prop} :
   image2 f s t ⊆ u ↔ ∀ (x ∈ s) (y ∈ t), f x y ∈ u :=
 forall_image2_iff
 
+lemma image2_subset_iff_left : image2 f s t ⊆ u ↔ ∀ a ∈ s, f a '' t ⊆ u :=
```

Comment:
but wait, there's more!
```suggestion
lemma image2_subset_iff_left : image2 f s t ⊆ u ↔ ∀ a ∈ s, (λ b, f a b) '' t ⊆ u :=
```

It's also worth putting a brief note about the deliberate eta expansion in the description; noting that it's good for symmetry, and it can help with rewriting when the lemmas are fully applied.